SAMLPortal/Misc/Helpers.cs

```diff
@@ -1,3 +1,5 @@
+using System.IO;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Mark this assembly with 'System.Runtime.InteropServices.ComVisibleAttribute'](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5343548)